lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -96,8 +96,8 @@ def specific_matcher(arg)
 
           def specific_matcher_option?(node, arg, matcher)
             attrs = CssSelector.attributes(arg).keys
-            return true if attrs.empty?
             return false unless replaceable_matcher?(node, matcher, attrs)
+            return true if attrs.empty?
```

Comment:
since attrs.all? will return true on an empty list, I guess we can just remove this line

Reply:
indeed it is, thank you so much. I removed it.